javabridge/tests/test_jutil.py

```diff
@@ -661,6 +661,12 @@ def test_12_03_jref_create_and_lock(self):
             'javabridge.unlock_jref(ref_id)', d, d)
         javabridge.unlock_jref(ref_self)
         self.assertRaises(KeyError, javabridge.redeem_jref, ref_self)
+
+class TestJutil(unittest.TestCase):
+    def test_with_stmt(self):
+        with javabridge.vm(run_headless=True):
```

Comment:
I think this is going to fail, unless it's the only test that's run. Unfortunately, the VM can't be restarted or duplicated (It's a limitation of the JVM, not Javabridge), so this will either fail or cause other tests to fail.